release-notes/release-notes54.md

```diff
@@ -79,6 +80,43 @@ The {{site.productname}} 5.4 release includes an accompanying release of the **P
 
 For information on the PowerPaste plugin, see: [PowerPaste plugin]({{site.baseurl}}/plugins/powerpaste/).
 
+## Accompanying premium self-hosted server-side component changes
```

Comment:
Should this be `Premium` as we used a capital above for `Premium Plugins`?

Reply:
premium is not used as a noun here, so that looks correct as is.  Learnt that from reviews :D